doc/install/manual_install.rst

```diff
@@ -13,50 +13,68 @@ Install via :code:`pip` or :code:`conda`
 MNE-Python requires Python version |min_python_version| or higher. If you
 need to install Python, please see :ref:`install-python`.
 
-- If you only need MNE-Python's core functionality including 2D plotting (but
-  **without 3D visualization**), install via :code:`pip`:
+Installing MNE-Python with all dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+We suggest to install MNE-Python into its own ``conda`` environment.
+
+The dependency stack is large and may take a long time (several tens of
+minutes) to resolve on some systems via ``conda``. We therefore highly
+recommend using ``mamba``, a ``conda`` replacement with a much faster solver.
+
+Run in your terminal:
```

Comment:
```suggestion
Make sure to be in your conda-environment ``base``.
Run in your terminal:
```
The mamba installation instructions emphasize that mamba is installed into the `base` environment.

Reply:
Actually it doesn't matter in which environment you are. The only important thing is that you install mamba 😅

Reply:
@marsipu I've just pushed a change that doesn't require the installation of `mamba`, but instead installs the experimental `libmamba` solver into the `base` environment. WDYT?

Reply:
Hm, installing `libmamba` in my environment resulted into a lengthy process of checking for incompatible packages.
Installing `mamba` was much quicker without this check. I don't know if that's only with my environment. What is the advantage of `libmamba` over 'mamba'?

Reply:
> What is the advantage of libmamba over 'mamba'?

Installing the `libmamba` solver avoids the need to install (and use) a tool that's new to users (i.e., `mamba`)

The solver is currently being tested and is likely to become the new default in future versions of `conda`

It'd odd it's not resolving the deps for you, as there are not many dependencies `libmamba` requires :\

For more info on this solver, see https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community